website/src/views/h3n2.ts

```diff
@@ -0,0 +1,176 @@
+import { type DataOrigin, dataOrigins } from '../types/dataOrigins.ts';
+import { CompareSideBySideStateHandler } from './pageStateHandlers/CompareSideBySidePageStateHandler.ts';
+
+const earliestDate = '1905-01-01';
+const hostField = 'hostNameScientific';
+
+class H3n2Constants implements OrganismConstants {
+    public readonly organism = Organisms.h3n2;
+    public readonly earliestDate = earliestDate;
+    public readonly mainDateField: string;
+    public readonly locationFields: string[];
+    public readonly lineageFilters: LineageFilterConfig[] = [
+        {
+            lapisField: 'cladeHA',
+            placeholderText: 'Clade HA',
+            filterType: 'text' as const,
+            initialValue: undefined,
+        },
+        {
+            lapisField: 'cladeNA',
+            placeholderText: 'Clade NA',
+            filterType: 'text' as const,
+            initialValue: undefined,
+        },
+    ];
+    public readonly useAdvancedQuery = false;
+    public readonly baselineFilterConfigs: BaselineFilterConfig[] = [
+        {
+            type: 'date',
+            dateRangeOptions: [
+                dateRangeOptionPresets.lastMonth,
+                dateRangeOptionPresets.last2Months,
+                dateRangeOptionPresets.last3Months,
+                dateRangeOptionPresets.last6Months,
+                dateRangeOptionPresets.lastYear,
+                { label: 'Since 2020', dateFrom: '2020-01-01' },
+                { label: '2010-2019', dateFrom: '2010-01-01', dateTo: '2019-12-31' },
+                { label: '2000-2009', dateFrom: '2000-01-01', dateTo: '2009-12-31' },
+                { label: 'Since 2000', dateFrom: '2000-01-01' },
+                { label: 'Before 2000', dateFrom: earliestDate, dateTo: '1999-12-31' },
+                { label: 'All times', dateFrom: this.earliestDate },
+            ],
+            earliestDate,
+            defaultDateRange: dateRangeOptionPresets.lastYear,
+            dateColumn: 'sampleCollectionDate',
+            label: 'Sample collection date',
+        },
+        {
+            lapisField: hostField,
+            placeholderText: 'Host',
+            type: 'text' as const,
+            label: 'Host',
+        },
+    ];
+    public readonly hostField: string = hostField;
+    public readonly authorsField: string | undefined;
+    public readonly authorAffiliationsField: string | undefined;
+    public readonly additionalFilters: Record<string, string> | undefined;
+    public readonly dataOrigins: DataOrigin[] = [dataOrigins.insdc];
+    public readonly accessionDownloadFields;
+    public readonly predefinedVariants = [
+        {
+            lineages: { cladeHA: '3C.2a1b.2a.2a.3a.1' },
+        },
+    ];
+
+    // Antiviral susceptibility mutations have been compiled here: https://www.who.int/teams/global-influenza-programme/laboratory-network/quality-assurance/antiviral-susceptibility-influenza/neuraminidase-inhibitor.
```

Comment:
Just making sure that this comment is not a copy-paste error and actually supposed to be here.

Reply:
Ah it should actually be a TODO - I thought I would add a mutation list in a separate PR as it still requires some work